### PR TITLE
Optionally output deciles charts

### PR DESCRIPTION
**Pull request by iaindillingham**
This PR follows #26. It involves refactoring several tests, and is opportunistic a couple of times (784307f8a2374c45ca20b59fff35b2bb45cab8a3, 1af03fd37bd4c5df8e49bcc320f35c3e242c93f0), so I'd suggest stepping through commit-by-commit.

The substantive commit is e8671ebe49f6ec5037f8382dc49c11f47631784c. Here, we add some extra configuration, which is discussed briefly in #4 and #18. This configuration allows us to optionally output deciles charts; it isn't especially useful now, but it will be with #18.

Keeping _README.md_ up-to-date is challenging with all reusable actions. However, to minimize the amount we have to remember to copy-and-paste from _project.yaml_, I've decided to invoke deciles-charts with default values for all configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,18 +77,21 @@ generate_deciles_charts:
...
       deciles_charts: output/deciles_chart_*.png
 ```
 
-| Configuration            | Default | Description                                                 |
-| ------------------------ | ------- | ----------------------------------------------------------- |
-| `show_outer_percentiles` | `false` | Show the top and bottom percentiles, as well as the deciles |
+| Configuration            | Description                                                 |
+| ------------------------ | ----------------------------------------------------------- |
+| `show_outer_percentiles` | Show the top and bottom percentiles, as well as the deciles |
+| `charts.output`          | Generate a deciles chart for each measure table             |
```
Comment (ccunningham101) [inline README.md:94]: ** For each measure table that matches the input-files regex

File: analysis/deciles_charts.py [hunk 2]
```diff
@@ -129,10 +139,11 @@ def main():
 
     for measure_table in get_measure_tables(input_files):
         measure_table = drop_zero_denominator_rows(measure_table)
-        chart = get_deciles_chart(measure_table, config)
-        id_ = measure_table.attrs["id"]
-        fname = f"deciles_chart_{id_}.png"
-        write_deciles_chart(chart, output_dir / fname)
+        if config["charts"]["output"]:
```
Comment (ccunningham101) [inline analysis/deciles_charts.py:142]: Right now if charts output is false then we do nothing? But in the future we might output the deciles table and just not the graphic, correct?

**Review by ccunningham101 — APPROVED** [review]

File: analysis/deciles_charts.py (hunk 2 above)
Comment (iaindillingham) [reply analysis/deciles_charts.py:142]: That's correct. We may wish to revisit outputs, depending on whether we find it easy but confusing to output nothing. For the moment, though, this is a small step towards outputting tables, charts, tables and charts, or neither tables nor charts 🙂

File: README.md (hunk 1 above)
Comment (iaindillingham) [reply README.md]: Good spot! I've changed and force-pushed 🙂

**Pull request merged**